climakitae/utils.py

```diff
@@ -374,3 +381,146 @@ def get_gwl_table(
         keys=list(models.keys()),
     )
     all_gw_levels2.to_csv("../data/gwl_1981-2010ref.csv")
+
+
+### utils for uncertainty notebooks
+class cmip_opt:
```

Comment:
Naming conventions for a class are to use camel case, so this should be named CmpiOpt